spec/asyncapi.md

```diff
@@ -565,7 +565,7 @@ Describes the operations available on a single channel.
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="channelItemObjectRef"></a>$ref | `string` | Allows for an external definition of this channel item. The referenced structure MUST be in the format of a [Channel Item Object](#channelItemObject). If there are conflicts between the referenced definition and this Channel Item's definition, the behavior is *undefined*.
+<a name="channelItemObjectRef"></a>$ref | `string` | Allows for an external definition of this channel item. The referenced structure MUST be in the format of a [Channel Item Object](#channelItemObject). If there are conflicts between the referenced definition and this Channel Item's definition, the behavior is *undefined*. <br><br>**Deprecated:** Usage of the `$ref` property has been deprecated.
```

Comment:
I think we should give a better explanation on the reason for deprecating. Like mentioning we deprecate it in favor of using `Reference Object` from the `Channels Object` instead of `Channel Item Object`. WDYT?

Reply:
If we say we deprecate in favor of `Reference Object` it may indicate that future versions will use `Reference Object`, but later versions of the spec might handle this quite differently. Best way to probably handle this is to have wording self-contained withing the version of the spec this RFC targets - and the sentence I put there was the only one I could come up with. I'm really open to suggestions here ;]

Reply:
I'm fine if others see it make sense 👍